pxr/usdImaging/usdImaging/cameraAdapter.cpp

```diff
@@ -259,9 +259,33 @@ UsdImagingCameraAdapter::Get(UsdPrim const& prim,
         cam.GetShutterCloseAttr().Get(&vShutterClose, time); // conversion n/a
         return vShutterClose;
     } else if (key == HdCameraTokens->exposure) {
-        VtValue v;
-        cam.GetExposureAttr().Get(&v, time); // conversion n/a
-        return v;
+        // we hijack the "exposure" token here in order to pass through the calculated
+        // exposure and make it backwards-compatible.
+        // existing clients will be expected it as logarithmic exposure, so we need
+        // to convert from a scalar multiplier to logarithmic exposure here.
+        // To get the original exposure attribute value, see "exposureCompensation"
+        // below
+        return VtValue(log2(cam.GetExposureScale(time)));
+    } else if (key == HdCameraTokens->exposureTime) {
+        VtValue vExposureTime;
+        cam.GetExposureAttr().Get(&vExposureTime, time); // conversion n/a
```

Comment:
I noticed that there's a bug here - it should be `GetExposureTimeAttr()`.

@tcauchois - given that you already have some outstanding internal changes, would you prefer I make an additional commit to this PR to fix this, or would that just complicate things for you?

Reply:
I've fixed it already internally, in the copy of the PR I pulled down.